__test__/unit/adapters/simple_interest_loan_adapter.spec.ts

```diff
@@ -151,48 +185,52 @@ describe("Simple Interest Terms Contract Interface (Unit Tests)", () => {
             });
         });
 
-        describe("...with valid expected repayment, amortization, and term length", () => {
+        describe("...with valid principal amount, interest rate, amortization, and term length", () => {
             describe("Scenario #1", () => {
                 test("should return correctly packed parameters", () => {
                     expect(simpleInterestLoanTerms.packParameters(defaultLoanParams)).toEqual(
-                        "0x00000000000000002ff62db077c0000010007000000000000000000000000000",
+                        "0x00000000002ff62db077c000000230f010007000000000000000000000000000",
```

Comment:
instead of hard coding these hex values, would prefer to use a function that generates them.

Reply:
Hmmm, not sure I agree.  I feel like we want to verify that the functions we use to pack those parameters are correct, and doing that by comparing those that are _also_ generated by a function in our test files introduces a situation in which both functions could be incorrect and the tests would still pass.  Could be convinced otherwise, though